jdaviz/configs/specviz/plugins/parsers.py

```diff
@@ -27,6 +28,10 @@ def specviz_spectrum1d_parser(app, data, data_label=None, format=None, show_in_v
         `~specutils.Spectrum1D.read` io method.
     spectrum_viewer_reference_name : str
         Reference name for the viewer
+    concat_by_file : bool
+        If True, concatenate the extensions within each spectrum file
+        passed to the parser and add a concatenated spectrum to the
+        data collection.
```

Comment:
is there any error/warning if the file does not contain multiple extensions?  It seems not from the code (which I think makes sense), but maybe the docstring should state that it is ignored otherwise?

Reply:
I've edited this docstring in the latest commit.